features/events/syslog/src/main/java/org/opennms/netmgt/syslogd/SyslogConnection.java

```diff
@@ -160,6 +161,13 @@ public SyslogProcessor call() {
         try {
             LOG.debug("Converting syslog message into event ({} bytes)", m_bytes.remaining());
 
+			if (!SyslogdIPMgrJDBCImpl.isDataBaseSync()) {
```

Comment:
Remove this code. We cannot reload the database state every time an event is processed. This will add lots of unnecessary overhead to the event handling.